tests/settings/utils_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2017  Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+
+from __future__ import absolute_import
+
+import unittest
+
+import mock
+
+from alot.settings import utils
+
+
+class TestResolveAtt(unittest.TestCase):
+
+    __patchers = []
+    fallback = mock.Mock()
+    fallback.foreground = 'some fallback foreground value'
+    fallback.background = 'some fallback background value'
+
+    @classmethod
+    def setUpClass(cls):
+        cls.__patchers.append(mock.patch('urwid.AttrSpec', mock.Mock(
```

Comment:
Thank you!
I will embrace the mantra :pray: although I didn't get the technical background yet.